src/aac/plugins/gen_design_doc/gen_design_doc_impl.py

```diff
@@ -47,7 +54,10 @@ def gen_design_doc(architecture_files: str, output_directory: str, template_file
     )
     write_generated_templates_to_file([generated_document], output_directory)
 
-    print(f"Wrote system design document to {os.path.join(output_directory, output_filespec)}")
+    result = PluginExecutionResult(plugin_name, PluginExecutionStatusCode.SUCCESS)
+    result.add_message(f"Wrote system design document to {os.path.join(output_directory, output_filespec)}")
+
+    return result
```

Comment:
Should add error handling, there are potential IO issues that could cause plugin failure.